tfhe/src/core_crypto/entities/seeded_lwe_packing_keyswitch_key.rs

```diff
@@ -456,3 +456,31 @@ impl<Scalar: UnsignedInteger, C: ContainerMut<Element = Scalar>> ContiguousEntit
     where
         Self: 'this;
 }
+
+impl<C: Container<Element = u64>> ParameterSetConformant for SeededLwePackingKeyswitchKey<C> {
+    type ParameterSet = PackingKeyswitchConformanceParams;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self {
+            data,
+            decomp_base_log,
+            decomp_level_count,
+            output_glwe_size,
+            output_polynomial_size,
+            ciphertext_modulus,
+            compression_seed: _,
+        } = self;
+
+        data.container_len()
+            == lwe_packing_keyswitch_key_input_key_element_encrypted_size(
+                *decomp_level_count,
+                *output_glwe_size,
+                *output_polynomial_size,
+            ) * parameter_set.input_lwe_dimension.0
```

Comment:
this has to be wrong following the same logic as the other seeded entities

Reply:
Fixed